Zend/tests/memory_reset_peak_usage.phpt

```diff
@@ -0,0 +1,25 @@
+--TEST--
+void memory_reset_peak_usage();
+--CREDITS--
+Patrick Allaert
```

Comment:
Usually, we don't add the `--CREDITS--` section to new tests, since that is already part of the commit and NEWS.